### DISCUSSION TITLE
Showing default fields in contact details on tickets

### DISCUSSION THREAD

**Discussion by LeahBreedenNyaruka** [nyaruka/rapidpro#4460] [Ideas]
Currently only featured custom contact fields are shown in contact details in tickets. May be helpful to include default fields such as 'language' and 'status'. Had a user ask about showing language specifically.

**Comment by ericnewcomer** [comment]
We added the fields tab to tickets. It wouldn't be hard to include the Details tab as well if we want to.

**Reply by ericnewcomer** [reply]
What say you @rowanseymour ?
